src/components/OrdersWidget/OrderRow.tsx

```diff
@@ -151,19 +162,50 @@ const Status: React.FC<Pick<Props, 'order' | 'isOverBalance' | 'transactionHash'
   ])
   const isFilled = useMemo(() => isOrderFilled(order), [order])
   // Display isLowBalance warning only for active and partial fill orders
-  const isLowBalance = isOverBalance && !isUnlimited && !isFilled && !isScheduled && !isPendingOrder && !isExpiredOrder
+  const isLowBalance =
+    isOverBalance &&
+    !isUnlimited &&
+    !isFilled &&
+    !isScheduled &&
+    !isActiveNextBatch &&
+    !isPendingOrder &&
+    !isExpiredOrder
 
   const pending = useMemo(() => isPendingOrder && <PendingLink transactionHash={transactionHash} />, [
     isPendingOrder,
     transactionHash,
   ])
 
+  // Dima's trick to force component update
+  const [, forceUpdate] = useSafeState({})
+
+  // Updates the component every second, if it has one of the states where it should happen
+  useEffect(() => {
+    let timer: null | NodeJS.Timeout = null
+
+    if (isActiveNextBatch || isFirstActiveBatch) {
+      timer = setInterval(() => forceUpdate({}), 1000)
+    } else if (!!timer) {
+      clearTimeout(timer)
+    }
+
+    return (): void => {
+      if (timer) clearTimeout(timer)
+    }
+  }, [forceUpdate, isActiveNextBatch, isFirstActiveBatch])
+
+  const secondsCountdown = formatSeconds(getSecondsRemainingInBatch())
```

Comment:
nit: This is not necesary unless you `pending && (isActiveNextBatch || isFirstActiveBatch)` but is executed for every row independently of those values

Would become simpler markup if you make a static function that returns the string of the status text?